crates/conformance-test-utils/src/main.rs

```diff
@@ -0,0 +1,34 @@
+use clap::Parser;
+use conformance_test_utils::utils::compute_starknet_address;
+use starknet::core::types::FieldElement;
+
+#[derive(Parser)]
+pub struct ComputeStarknetAddressArgs {
+    #[arg(long)]
+    #[arg(short = 'k')]
+    #[arg(help = "Set the kakarot address.")]
+    pub kakarot_address: FieldElement,
+    #[arg(long)]
+    #[arg(short = 'a')]
+    #[arg(help = "Set the account proxy class hash.")]
+    pub account_proxy_class_hash: FieldElement,
+    #[arg(long)]
+    #[arg(short = 'e')]
+    #[arg(help = "Set the EVM address.")]
+    evm_address: FieldElement,
```

Comment:
For now since we have only one command, I'm putting it here. Eventually I'll separate them when we have more commands